pages/posts/article.jsx

```diff
@@ -97,13 +97,13 @@ const articleData = {
 };
 
 export default () => {
-  return (
     <Layout>
       <Head>
         <title>Foo bar</title>
       </Head>
 
```

Comment:
The return statement is missing parentheses, causing incorrect JSX structure.

Suggestion:
```javascript
return (
  <Layout>
    <Head>
      <title>Foo bar</title>
    </Head>
    <Article {...articleData} />
  </Layout>
);
```